src/popup/components/InputPassword.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <InputField
+    class="input-password"
+    v-bind="$attrs"
+    :model-value="modelValue"
+    :type="isPasswordVisible ? 'text' : 'password'"
+    @update:modelValue="$emit('update:modelValue', $event)"
+  >
+    <template #after>
+      <component
+        :is="isPasswordVisible ? EyeClosedIcon : EyeIcon"
+        v-if="!hideEyeIcon"
+        class="eye-icon"
+        @click="toggleVisibility"
+      />
+    </template>
+    <template #label-after>
+      <span
+        v-if="showPasswordStrength && modelValue"
+        :class="['password-strength', passwordStrength.toLowerCase()]"
```

Comment:
I suggest to set `password-strength` class in static way so IDE can catch this (at least VSCode) and provide the details about used/unused classes.